packages/playground/package.json

```diff
@@ -18,5 +19,12 @@
     "react": "^18.2.0",
     "react-dom": "^18.2.0"
   },
-  "license": "MPL-2.0"
+  "license": "MPL-2.0",
+  "devDependencies": {
```

Comment:
I would like to keep the devDeps here. I do not think putting all devDeps in the root package.json is a good practice. Each package should be as autonomous as possible. The root packagejson.devDeps shall contains the devDeps for the command ran from the root.

Reply:
Vite is in the root package, and no build-toolchain-related packages are placed in the `playground`. That's why I would suggest reusing the toolchain across the monorepo. It can be painful to manage such kind of "autonomous".

Reply:
But the deps and devDeps in the root-level are phantom dependencies to all the sub-packages 🤔 It's really dangerous IMO. We only can find that by code review.

Reply:
We don't have any `dependencies` at the root level, we are only sharing the toolchain in the root level `devDependencies`. IMO it really doesn't make much sense if one has to configure redundant `devDependencies` in every sub-package. I think that's also how Vue is doing ( https://github.com/vuejs/core/tree/main/packages ).

Reply:
Vite is doing it in another way 😬 https://github.com/vitejs/vite/search?l=JSON&q=%22fast-glob%22. I admit that redundant devDep is painful and I also hate phantom dependency. Since there's no perfect strategy, I would like to move it to the root-level. Maybe we can find a better solution in the future 🤔.

Reply:
Done, have a look.